### PR TITLE
jQuery updated

### PR DESCRIPTION
**Pull request by mohit-003**
For issue #111 . jQuery version changed to 3.4.1

### CODE REVIEW

**Comment by mohit-003** [comment]
@sal2701 Done!

File: _includes/fsm.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
     <script src="../assets/js/ModuleCheckpoints.js"></script>
     <script
-            src="http://ajax.aspnetcdn.com/ajax/jQuery/jquery-1.10.2.min.js"></script>
+            src="https://ajax.aspnetcdn.com/ajax/jQuery/jquery-3.4.1.min.js"></script>
```
Comment (sal2701) [inline _includes/fsm.html]: @mohit-003  Can you do something so that you don't have to mention http or https explicitly?

Comment (mohit-003) [reply _includes/fsm.html]: > @mohit-003 Can you do something so that you don't have to mention http or https explicitly?

Yes done!

**Comment by mohit-003** [comment]
Yes done👍🏻

**Comment by sal2701** [comment]
@tachyons @satu0king @ankitkataria Please review this PR and merge it as it solves some issues which need to be addressed immediately. Changes made here are as follows
* Changed jquery from slim version to normal version to fix Flip-Flop Animations.
* Corrected path of Font-Awesome files.
* Changed file path to local to fix http https error by Netlify.

**Pull request merged**